addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3315,11 +3420,21 @@ class SnippetsMenu extends Component {
                     'drop-near': option['drop-near'] ? canDrop(option['drop-near'].all()) : false,
                     'drop-in': option['drop-in'] ? canDrop(option['drop-in'].all()) : false,
                 };
-                check = (cache[k]['drop-near'] || cache[k]['drop-in']);
+
+                const hasDropNear = cache[k]['drop-near'];
+                const hasDropIn = cache[k]['drop-in'];
+                check = isSnippetStructure ? hasDropIn : hasDropNear || hasDropIn;
```

Comment:
Could you add a comment to explain why the `isSnippetStructure ?` check is needed? At first sight, it seems the condition would already be covered by the `||`. (If I got it right, it's to prevent the use of groups for hybrid snippets that would only have a drop-near, and thus only be droppable inside another snippet?)

Reply:
> prevent the use of groups for hybrid snippets

Yes, that's exactly why. I added a comment to explain, thanks.